src/config/env.json

```diff
@@ -1,33 +1,47 @@
 {
   "default": {
+    "PORT": 3010,
+    "SERVICE_NAME": "guardianship_agency",
+    "MANAGER_TYPE": "K8S",
     "DD_AGENT_HOST": "datadog-statsd.datadog",
```

Comment:
I liked to keep the alphabetical.  It makes it easier to find as the list grows.  thoughts?

Reply:
Ah that makes sense - will change